packages/react/src/forms/Checkbox/Checkbox.module.css

```diff
@@ -5,6 +5,11 @@
   position: relative;
 }
 
+[class*='FormControl'] .Checkbox-wrapper,
```

Comment:
This might be overkill but I wasn't sure if we wanted to always apply the grid-row to the wrapper for the input or only do this when its parent is a form control element?